modules/recording/recording/dashboard.tsx

```diff
@@ -73,16 +73,50 @@ const Dashboard = () => {
     }
   };
 
+  const startVisualization = () => {
+    if (!analyserRef.current && audioRecorderRef.current) {
```

Comment:
Use `if-negative return` 

I mean ` if (analyserRef.current || audioRecorderRef.current) { return; } ` 

For all the nested ifs. 

The if chain shall be flat.